### PR TITLE
Kevin polish 4

### PR DESCRIPTION
**Pull request by kadst43**

### CODE REVIEW

File: source/triggers/clauses/clause_IsAtPosition.js [hunk 1]
```diff
@@ -48,6 +50,8 @@ this.onGenerated = function( params, generator, payload ) {
         return false;
     }
 
+    this.useAxisOffset = !!params[ 2 ];
```
Comment (AmbientOSX) [inline source/triggers/clauses/clause_IsAtPosition.js]: What's the wisdom on the !! here? Asking purely for academic purposes :)

File: source/view/blocks.js [hunk 2]
```diff
@@ -1662,377 +1662,377 @@ Blockly.JavaScript['triangle_operations_dilate_auto_fixed'] = function( block )
...
+        Blockly.FieldTextInput.numberValidator), "CX")
+        .appendField(",")
+        .appendField(new Blockly.FieldTextInput('0',
+        Blockly.FieldTextInput.numberValidator), "CY")
+        .appendField(")");
+    this.setInputsInline(false);
+    this.setPreviousStatement(true, 'INPUT');
+    this.setNextStatement(true);
+    this.setColour(195);
+    this.setTooltip('');
```
Comment (AmbientOSX) [inline source/view/blocks.js]: Be sure to check the other blocks for the correct tooltip formatting here when the tooltip is ultimately added.

**Comment by AmbientOSX** [comment]
A few non-blocking questions then :+1:

File: source/triggers/clauses/clause_IsAtPosition.js (hunk 1 above)
Comment (kadst43) [inline source/triggers/clauses/clause_IsAtPosition.js]: It forces it to be a Boolean.  It's really not necessary in this case, but I felt like it added clarity (and prevents anybody from abusing the value, I suppose).

**Pull request merged**